_posts/blog/2017-03-10-public_speaking_blank_slides.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "A Public Speaking Experiment: Blank Slides"
+description:
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: My talk at the International Women’s Day event, organised by Women Techmakers gave me the chance to share my story about public speaking but also to try something new - using blank slides.
+tags: [Product, Management, Data, Data Science, User, Feedback, ProductManagement]
+image:
+date: 2017-03-08T00:00:00-04:00
+---
+My talk at the <a href="https://www.womentechmakers.com/iwd17" target="blank">International Women’s Day event</a>, organised by <a href="https://www.womentechmakers.com/" target="blank">Women Techmakers</a> gave me the chance to share my story about public speaking but also to try something new: using **blank slides**.
+
+<center>
+<picture>
+	<a href="/images/blog/public_speaking_blank_slides/google_slides.png"><img src="/images/blog/public_speaking_blank_slides/google_slides.png" alt="Public Speaking Blank Slides"></a>
+	<figcaption>Screenshot from my Google Slides</figcaption>
+</picture>
+</center>
+
+The time allocated for my talk was 25min + 5min for Q&A. Since this was a non-tech talk that didn’t require much visual support, I decided to combine text slides, with blank slides. You can find the final slides <a href="https://www.slideshare.net/FlorinaMuntenescu/tech-talks-you-do-have-something-to-say-why-and-how-to-start" target="blank">here</a>.
+
+#### My strategy was this:
+
+* Use **blank slides** for the moments when I’m telling **my story**. Like this, the focus of the audience is on what I’m *saying*.
+* Use **text slides** only for **quotes** and **main take-aways**. These would only appear in the key moments of my presentation and the audience should focus on them. The effect of the take-aways is now amplified, since these are the only parts that the audience both *hears and sees*.
+
+Usually in my tech talks, I tend to have one slide per idea. So I kept the same approach now, even if my slides were blank. This also gives me a sign of when I should have a small break in my speaking flow.
+
+#### There are a few possible downsides:
+
+* The **IT audience** is used to talks where they always have a visual aid: slides with text and/or images so having a black screen all of the sudden can be a surprise. I saw this effect when rehearsing my talk with a friend. But, after the initial **surprise**, I could see that my strategy is working: he was only looking at the slides when there was something there and the rest of the time, his attention was on what I was saying.
```

Comment:
I got feedback from 3 people and all of them used the word "surprised" because it was something unexpected.